Stack/Opc.Ua.Core/Schema/ApplicationConfiguration.cs

```diff
@@ -868,14 +869,31 @@ public CertificateStoreIdentifier RejectedCertificateStore
             set { m_rejectedCertificateStore = value; }
         }
 
+        /// <summary>
+        /// Gets or sets a value indicating how many certificates are kept
+        /// in the rejected store before the oldest is removed.
+        /// </summary>
+        /// <remarks>
+        /// This value can be set by applications.
+        /// The number of certificates to keep in the rejected store before it is updated.
+        /// <see langword="0"/> to keep all rejected certificates.
+        /// A negative number to keep no history.
+        /// </remarks>
+        [DataMember(IsRequired = false, EmitDefaultValue = false, Order = 8)]
```

Comment:
It's always good to add the new elements to the end of the list, not to break backwards compatibility. - In case a config file for a new stack is used in an older stack, it would create problems.

Reply:
I wanted it closer to where it is used , aka rejected store